robot/Crisis-Management/resources/CrisisManagement.py

```diff
@@ -51,6 +53,75 @@ def __init__(self, debug=False):
         logging.getLogger('requests.packages.urllib3.connectionpool').setLevel(logging.WARN)
         self._init_locators()
 
+    @capture_screenshot_on_error
+    def select_value_from_dropdown(self,dropdown,value):
+        """Select given value in the dropdown field"""
+        elelocator = "//div[contains(@class,'forcePageBlockSectionRow')]/div[contains(@class,'forcePageBlockItem')]/div[contains(@class,'slds-hint-parent')]/div[@class='slds-form-element__control']/div[.//span[text()='{}']][//div[contains(@class,'uiMenu')]//a[@class='select']]"
+
+        locator = elelocator.format(dropdown)
+        self.selenium.scroll_element_into_view(locator)
+        self.selenium.get_webelement(locator).click()
+        #self.wait_for_locator('popup')
+        self.selenium.click_link(value)
+
+    def check_if_element_exists(self, xpath):
+        elements =self.selenium.get_element_count(xpath)
+        return True if elements > 0 else False
+
+    def get_url_formatted_object_name(self,name):
+        """Returns a map with BaseURl and the namespace formatted object name"""
+        out = {}
+        base_url = self.cumulusci.org.lightning_base_url
+        object_name = "{}{}".format(self.cumulusci.get_namespace_prefix(), name)
+        out['baseurl'] = base_url
+        out['objectname'] = object_name
+        return out
+
+    @capture_screenshot_on_error
+    def populate_modal_form(self,**kwargs):
+        """Populates modal form with the field-value pairs
+        supported keys are any input, textarea, lookup, checkbox, date and dropdown fields"""
+    
+        for key, value in kwargs.items():
+            locator = cm_lex_locators["modal-form"]["label"].format(key)
+            if self.check_if_element_exists(locator):
+                ele=self.selenium.get_webelements(locator)
+                for e in ele:
+                    classname=e.get_attribute("class")
+                    #                     print("key is {} and class is {}".format(key,classname))
+                    if "Lookup" in classname and "readonly" not in classname:
+                        self.salesforce.populate_lookup_field(key,value)
+                        print("Executed populate lookup field for {}".format(key))
```

Comment:
Same comment as above for print statement.

Reply:
Added this just for debugging purposes